modules/govuk_rabbitmq/manifests/queue_with_binding.pp

```diff
@@ -65,10 +70,12 @@
     routing_key      => $routing_key,
     arguments        => {},
   }
-
-  govuk_rabbitmq::monitor_consumers {"${title}_${amqp_queue}_consumer_monitoring":
-    ensure            => $ensure,
-    rabbitmq_hostname => 'localhost',
-    rabbitmq_queue    => $amqp_queue,
+  
+  if $monitor_consumers {
+    govuk_rabbitmq::monitor_consumers {"${title}_${amqp_queue}_consumer_monitoring":
+      ensure            => $ensure,
```

Comment:
I've checked, and this class just involves Icinga configuration, which I believe is fully managed, so making the configuration itself conditional should be OK (rather than changing the `ensure` value).